include/libtorrent/announce_entry.hpp

```diff
@@ -159,40 +194,13 @@ namespace libtorrent {
 		bool deprecated_send_stats:1;
 #endif
 
-		// internal
-		bool triggered_manually:1;
-
 		// reset announce counters and clears the started sent flag.
 		// The announce_entry will look like we've never talked to
 		// the tracker.
 		void reset();
 
-		// updates the failure counter and time-outs for re-trying.
-		// This is called when the tracker announce fails.
-		void failed(int backoff_ratio, seconds32 retry_interval = seconds32(0));
-
-#ifndef TORRENT_NO_DEPRECATE
-		// deprecated in 1.0
-		TORRENT_DEPRECATED
-		bool will_announce(time_point now) const
-		{
-			return now <= next_announce
-				&& (fails < fail_limit || fail_limit == 0)
-				&& !updating;
-		}
-#endif
-
-		// returns true if we can announce to this tracker now.
-		// The current time is passed in as ``now``. The ``is_seed``
-		// argument is necessary because once we become a seed, we
-		// need to announce right away, even if the re-announce timer
-		// hasn't expired yet.
-		bool can_announce(time_point now, bool is_seed) const;
```

Comment:
for API backwards compatibility, do you think there's a reasonable way to add this and and ``is_working()`` back as deprecated?